test/suites/fastq/resources/mtdna_fazzini_gs.cfg

```diff
@@ -0,0 +1,13 @@
+params {
+    cram.call_str = false
+    cram.call_sv = false
+    cram.call_cnv = false
+    vcf.filter.classes = "LQ,B,LB,VUS,LP,P"
+    vcf.filter_samples.classes = "U1,U2,U3"
+}
+
+process {
+    withLabel: 'deepvariant_call' {
+        time = '9h'
```

Comment:
this indicates an issue in the default config of vip. is it still required now that a .bed file is used?